components/ui/Tag.jsx

```diff
@@ -0,0 +1,9 @@
+function Tag(props) {
+  return (
+    <button className="m-auto h-fit rounded-4.75 bg-light-grey px-2 text-grey">
```

Comment:
This component is missing some CSS properties considering those on figma design

- [ ] Padding on y
- [ ] font-weight
- [ ] font-style
- [ ] font-family
- [ ] letter spacing